src/ofp_arp.c

```diff
@@ -608,13 +611,23 @@ enum ofp_return_code ofp_arp_save_ipv4_pkt(odp_packet_t pkt, struct ofp_nh_entry
 
 	odp_rwlock_write_lock(&shm->arp.set[set].table_rwlock);
 
-#if (ARP_SANITY_CHECK)
-	newarp = arp_lookup(set, &key);
-       if (newarp && newarp->is_valid)
-		OFP_ERR("ARP Entry failed the sanity check!");
-#endif
+	if (is_link_local) {
+		odp_rwlock_t *lock = &shm->arp.set[set].table_rwlock;
```

Comment:
`odp_rwlock_t *lock` could be moved to upper level so all lock/unlock functions could use the same variable making the code a bit easier to read.

Reply:
OK